packages/scene-composer/package.json

```diff
@@ -50,7 +50,7 @@
     "fix": "eslint --fix .",
     "test": "jest --config jest.config.ts --coverage --silent",
     "test:dev": "jest --config jest.config.ts --coverage",
-    "posttest": "jest-coverage-ratchet",
+    "test:update-threashold": "jest-coverage-ratchet",
```

Comment:
to avoid the no un-commited change failure https://github.com/awslabs/iot-app-kit/actions/runs/4380489461/jobs/7667612621